spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/convert/ConverterAwareMappingSpannerEntityWriter.java

```diff
@@ -286,8 +288,8 @@ private boolean attemptSetIterableValue(Iterable<Object> value,
 		if (!valueSet) {
 			for (Class<?> targetType : iterablePropertyType2ToMethodMap.keySet()) {
 				if (this.writeConverter.canConvert(innerType, targetType)) {
-					BiConsumer<ValueBinder<?>, Iterable> toMethod = iterablePropertyType2ToMethodMap
-							.get(targetType);
+					BiConsumer<ValueBinder<Mutation.WriteBuilder>, Iterable> toMethod =
```

Comment:
@meltsufin I removed the WriteBuilder type here because the ValueBinder type is actually used with several things including a the SQL param binder. Indeed we do use this same map of ValueBinder functions with both.

Reply:
Yeah, feel free to revert it. The type parameterization of `ValueBinder` is weird because its only purpose is the support for a fluent API.